src/common/utils/email/mail.go

```diff
@@ -106,7 +53,11 @@ func sendMailWithTLS(m Mail, auth smtp.Auth, content []byte) error {
 		return err
 	}
 
-	_, err = w.Write(content)
+	template := "From: %s\r\nTo: %s\r\nSubject: %s\r\nMIME-version: 1.0;\r\nContent-Type: text/html; charset=\"UTF-8\"\r\n\n%s\r\n"
```

Comment:
why is \r\n needed?

and btw, I think you can use `` to quote a string so "\n" is not needed.